members_club/data/sale_subscription_plan.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="sale_subscription_plan_3" model="sale.subscription.plan">
```

Comment:
Remove these 2 records, and use standard ones (sale_subscription.subscription_plan_month and sale_subscription.subscription_plan_year)